README.md

```diff
@@ -1,43 +1,141 @@
-# Official Cognite Data Fusion project templates
+# Official Cognite Data Fusion Project configuration templates
 
-The CDF Toolkit is a command-line interface (`cdf-tk`) used for configuring and administrating Cognite Data
-Fusion (CDF) projects.
-It supports three different modes of operation:
+> Configure Cognite Data Fusion quickly, consistently, traceably, and repeatably.
+ 
+> [!WARNING]
+> The templates and the `cdf-tk` tool are currently in **ALPHA**. The scope in alpha is on Asset
+Performance Management focused on Infield (Digital Operator Rounds). The templates and tooling
+will be continously improved throughout moving towards beta and general availability.
 
-1. As an **interactive command-line tool** used alongside the Cognite Data Fusion web application to retrieve and
```

Comment:
Why don't you want the description of the three different use cases here?

Reply:
For brevity. I think of this as the stepwise guide, TLDR version